packages/zowe-explorer-api/src/vscode/ui/TableViewProvider.ts

```diff
@@ -60,7 +60,7 @@ export class TableViewProvider implements WebviewViewProvider {
      * Provide a table view to display in the "Zowe Resources" view.
      * @param tableView The table view to prepare for rendering
      */
-    public setTableView(tableView: Table.Instance | null): void {
+    public async setTableView(tableView: Table.Instance | null): Promise<void> {
```

Comment:
same as previous comment 😋 
Do we want a changelog entry to capture when this was made asynchronous?

Reply:
addressed in b07a19a